packages/core/src/codewhisperer/util/authUtil.ts

```diff
@@ -58,9 +58,9 @@ export const isValidCodeWhispererCoreConnection = (conn?: Connection): conn is C
 /** For Builder ID only, if using IdC then use {@link isValidAmazonQConnection} */
 export const isValidCodeWhispererChatConnection = (conn?: Connection): conn is Connection => {
```

Comment:
In your old PR `isValidAmazonQConnection` was modified, now it's `isValidCodeWhispererChatConnection`?

Reply:
fixed